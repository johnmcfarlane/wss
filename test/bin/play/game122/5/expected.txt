 81  3,1  | PIQU
 49  1,3  - QUO
 48  9,1  - OUF
 46  1,3  - OUF
 46  1,3  - SUF
 44  7,4  | FPS
 40  2,5  | OPUSES
 39  3,1  | FIP
 39  3,1  - FU
 39  3,1  | QIS
 38  7,4  | OPS
 36  2,6  | FUSES
 36  2,6  | PUSES
 36  2,6  | SPUES
 36  2,6  | SUPES
 36  2,6  - UPJET
 35  3,1  - PO
 33  3,1  | PIOUS
 33  3,1  | QI
 33  3,1  | SIPS
 33  9,1  - UFO
 33  9,1  - UFS
 33  9,1  - UPO
 33  9,1  - UPS
 32  2,6  | POSES
 31  1,3  - PUS
 31  7,5  | SSP
 30  7,4  | FP
 30  3,1  | PISOS
 30  3,1  | PIUS
 30  2,7  | QUESO
 30  3,1  | SIP
 29  7,5  | SSU
 28  2,7  | QUES
 28  7,6  | SUQS
 27  9,1  - OPS
 27  3,1  | PIU
 27  1,3  - POS
 27  9,1  | SsP
 26  9,3  - OUF
 26  7,6  | SOUPS
 26  7,6  | SUQ
 25  3,7  | FOT
 25  7,6  | SOUP
 24  2,9  | ESQ
 24  7,4  | OP
 24  3,1  | PISO
 24  3,1  | PISS
 24  2,8  | SEQ
 24  9,1  - UP
 23  9,3  - OPS
 22  1,3  - FU
 22  7,6  | SUPS
 22  1,3  - SUS
 22  4,1  | UMP
 21  2,3  - OF
 21  3,1  | PIS
 21  7,6  | SOPS
 21  3,9  | TUP
 21  3,4  - UPNESS
 20  2,2  | LOUPS
 20  3,8  | OTUS
 20  2,4  | POUSSE
 20  2,4  | SPOUSE
 20  7,6  | SUF
 20  7,6  | SUP
 19  8,2  | KOPS
 19  7,6  | SOP
 19  7,6  | SOUS
 19  7,6  | SSP
 19  5,5  - ZAPUS
 18  8,2  | KOP
 18  9,1  - OF
 18  9,1  - OP
 18  7,4  | OS
 18  1,3  - PO
 18  1,3  - SOS
 18  3,9  | TOP
 18  3,9  | TPS
 17  3,1  - SO
 17  7,6  | SOU
 17  7,6  | SSU
 17  7,6  | SUS
 16  2,8  | FEUS
 16  2,5  | FOSSE
 16  2,6  | FUSE
 16  2,5  | POSSE
 16  2,7  - QUOR
 16  7,6  | SOS
 16  2,7  | SPEOS
 16  2,6  | SPUE
 16  2,6  | SUPE
 16  5,5  - ZAPS
 16  9,2  | sUQS
 15  3,1  | FI
 15  4,1  | OMS
 15  3,1  | PI
 15  7,6  | SO
 15  9,1  | SsU
 15  9,1  - US
 15  5,5  - ZAP
 14  2,8  | FESS
 14  2,8  | FEU
 14  2,7  | FOES
 14  2,7  | OPES
 14  2,8  | PESO
 14  2,8  | PESS
 14  2,6  | POSE
 14  2,8  | SEPS
 14  2,6  | SOPE
 14  9,2  | sUQ
 13  4,6  - JETS
 13  2,2  | LOQ
 13  2,7  | QUE
 13  9,2  | sOUPS
 12  2,9  | EFS
 12  2,1  | FLOPS
 12  3,2  | IPSO
 12  8,2  | KOU
 12  8,2  | KUSSO
 12  2,8  | PES
 12  2,8  | SEP
 12  3,1  | SIS
 12  1,7  - SOPOR
 12  2,5  | SOUSE
 12  3,9  | TOU
 12  5,5  - ZAS
 12  9,2  | sOUP
 11  2,1  | FLOP
 11  3,4  - PONES
 11  2,3  - US
 10  2,9  | EF
 10  3,4  - FONE
 10  3,7  - FOR
 10  3,2  | IFS
 10  3,2  | IPS
 10  8,2  | KOSS
 10  2,1  | PLF
 10  3,4  - PONE
 10  2,10 - SO
 10  2,7  | SUES
 10  3,4  - SUNE
 10  2,7  | USES
 10  9,2  | sUPS
  9  2,1  | FLOSS
  9  2,1  | FLUS
  9  3,2  | IF
  9  8,2  | KOS
  9  2,2  | LOUP
  9  9,1  - OS
  9  2,3  - OS
  9  2,1  | PLUS
  9  2,1  | SLOPS
  9  2,1  | SLUP
  9  1,3  - SO
  9  1,7  - SPOOR
  9  3,9  | TOS
  9  3,9  | TSS
  9  1,9  | VSS
  8  2,9  | ESU
  8  2,1  | FLU
  8  2,2  | LOPS
  8  9,3  - OF
  8  9,3  - OP
  8  2,7  | OSES
  8  2,6  | OSSE
  8  2,1  | PLU
  8  2,7  - POOR
  8  2,1  | SLOP
  8  3,4  - SONES
  8  9,2  | sOPS
  8  9,2  | sUF
  8  9,2  | sUP
  8  9,2  | sUSSO
  7  2,1  | FLO
  7  2,2  | LOF
  7  2,2  | LOP
  7  5,4  - NEF
  7  5,4  - NESS
  7  2,1  | OLP
  7  4,4  - ONES
  7  3,4  - SONE
  7  1,7  | SOV
  7  9,2  | sUSS
  6  2,9  | ESS
  6  2,7  | FOE
  6  2,8  | OES
  6  3,1  | OI
  6  4,4  - ONE
  6  2,7  | OPE
  6  2,7  | POE
  6  3,1  | SI
  6  4,1  | UM
  6  9,1  | Us
  6  9,2  | sOP
  6  9,2  | sOUS
  6  9,2  | sSP
  5  2,8  | FE
  5  3,2  | IOU
  5  2,2  | LOSS
  5  2,2  | LOU
  5  2,2  | LUO
  5  4,1  | OM
  5  9,3  - OS
  5  2,8  | PE
  5  9,2  | sOSS
  5  9,2  | sUS
  4  2,9  | EO
  4  2,9  | ES
  4  2,2  | LU
  4  5,4  - NEO
  4  2,7  | SUE
  4  2,7  | USE
  4  9,2  | sOU
  4  9,2  | sSU
  3  3,2  | IO
  3  3,2  | IS
  3  2,2  | LO
  3  2,7  | OSE
  3  9,1  | Os
  3  2,7  | SOE
  3  9,2  | sOS
  2  2,8  | OE
  2  3,9  | TO
  2  9,2  | sO
