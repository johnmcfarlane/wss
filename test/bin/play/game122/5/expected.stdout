 40  2,5  | OPUSES
 39  3,1  | QIS
 36  2,6  | FUSES
 36  2,6  | PUSES
 36  2,6  | SPUES
 36  2,6  | SUPES
 35  3,1  - PO
 33  3,1  | PIOUS
 33  3,1  | QI
 33  3,1  | SIPS
 33  9,1  - UPO
 33  9,1  - UPS
 32  2,6  | POSES
 30  3,1  | PISOS
 30  3,1  | SIP
 28  7,6  | SUQS
 27  9,1  - OPS
 27  3,1  | PIU
 26  7,6  | SOUPS
 26  7,6  | SUQ
 25  7,6  | SOUP
 24  3,1  | PISO
 24  3,1  | PISS
 24  9,1  - UP
 23  9,3  - OPS
 22  7,6  | SUPS
 22  4,1  | UMP
 21  2,3  - OF
 21  3,1  | PIS
 21  7,6  | SOPS
 21  3,9  | TUP
 20  2,2  | LOUPS
 20  2,4  | SPOUSE
 20  7,6  | SUP
 19  8,2  | KOPS
 19  7,6  | SOP
 19  7,6  | SOUS
 18  8,2  | KOP
 18  9,1  - OF
 18  9,1  - OP
 18  1,3  - PO
 18  1,3  - SOS
 18  3,9  | TOP
 17  3,1  - SO
 17  7,6  | SOU
 16  2,8  | FEUS
 16  2,5  | FOSSE
 16  2,6  | FUSE
 16  2,5  | POSSE
 16  7,6  | SOS
 16  2,6  | SPUE
 16  2,6  | SUPE
 16  5,5  - ZAPS
 16  9,2  | sUQS
 15  3,1  | FI
 15  4,1  | OMS
 15  3,1  | PI
 15  7,6  | SO
 15  9,1  - US
 15  5,5  - ZAP
 14  2,8  | FESS
 14  2,8  | FEU
 14  2,7  | FOES
 14  2,7  | OPES
 14  2,8  | PESO
 14  2,6  | POSE
 14  9,2  | sUQ
 13  4,6  - JETS
 13  9,2  | sOUPS
 12  2,9  | EFS
 12  2,1  | FLOPS
 12  8,2  | KUSSO
 12  2,8  | PES
 12  3,1  | SIS
 12  1,7  - SOPOR
 12  2,5  | SOUSE
 12  5,5  - ZAS
 12  9,2  | sOUP
 11  2,1  | FLOP
 11  3,4  - PONES
 10  2,9  | EF
 10  3,7  - FOR
 10  3,2  | IFS
 10  8,2  | KOSS
 10  3,4  - PONE
 10  2,10 - SO
 10  2,7  | SUES
 10  2,7  | USES
 10  9,2  | sUPS
  9  2,1  | FLOSS
  9  2,1  | FLUS
  9  3,2  | IF
  9  8,2  | KOS
  9  2,2  | LOUP
  9  9,1  - OS
  9  2,3  - OS
  9  2,1  | PLUS
  9  2,1  | SLOPS
  9  1,3  - SO
  9  1,7  - SPOOR
  8  2,1  | FLU
  8  2,2  | LOPS
  8  9,3  - OF
  8  9,3  - OP
  8  2,7  | OSES
  8  2,7  - POOR
  8  2,1  | SLOP
  8  3,4  - SONES
  8  9,2  | sOPS
  8  9,2  | sUP
  7  2,2  | LOP
  7  5,4  - NEF
  7  5,4  - NESS
  7  4,4  - ONES
  7  3,4  - SONE
  7  9,2  | sUSS
  6  2,9  | ESS
  6  2,7  | FOE
  6  2,8  | OES
  6  3,1  | OI
  6  4,4  - ONE
  6  2,7  | OPE
  6  3,1  | SI
  6  4,1  | UM
  6  9,1  | Us
  6  9,2  | sOP
  6  9,2  | sOUS
  5  2,8  | FE
  5  2,2  | LOSS
  5  4,1  | OM
  5  9,3  - OS
  5  2,8  | PE
  4  2,9  | ES
  4  2,7  | SUE
  4  2,7  | USE
  4  9,2  | sOU
  3  3,2  | IS
  3  2,2  | LO
  3  2,7  | OSE
  3  9,1  | Os
  3  9,2  | sOS
  2  2,8  | OE
  2  3,9  | TO
  2  9,2  | sO
